ts/Accessibility/Components/InfoRegionsComponent.ts

```diff
@@ -487,6 +487,8 @@ class InfoRegionsComponent extends AccessibilityComponent {
         // Sections are wrapped to be positioned relatively to chart in case
         // elements inside are tabbed to.
         sectionDiv.style.position = 'relative';
+        sectionDiv.style.height = '0px';
```

Comment:
Two things here:
1. If we set 0px, this may cause some browsers to remove the element from the accessibility tree. Can we do 1px? 
2. Do we need to consider styled mode?

Reply:
Changed solution (removed those lines).